packages/dso-toolkit/src/components/plekinfo-card/plekinfo-card.args.ts

```diff
@@ -5,18 +5,25 @@ import { noControl } from "../../storybook";
 import { PlekinfoCard, PlekinfoWijzigactie } from "./plekinfo-card.models.js";
 import { Renvooi } from "../renvooi/renvooi.models.js";
 import { Label } from "../label/label.models.js";
+import { Button } from "../button";
+import { Toggletip } from "../toggletip/toggletip.models";
+import { SlideToggle } from "../slide-toggle";
 
 export interface PlekinfoCardArgs {
   label: Renvooi | string;
   href: string;
+  targetBlank: boolean;
+  interactions: Array<Button | Label | Toggletip<never> | SlideToggle>;
```

Comment:
Slechts de `SlideToggle` gaat in het `interactions` slot, maar aangezien het ook de enige interaction is die in dat slot gaat kunnen we die net zo goed hernoemen naar interaction en hoeft het ook geen Array meer te zijn.